src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs

```diff
@@ -28,7 +28,8 @@ public class FrameRenderer : CardView, IPlatformViewHandler
 				[Frame.CornerRadiusProperty.PropertyName] = (h, _) => h.UpdateCornerRadius(),
 				[Frame.BorderColorProperty.PropertyName] = (h, _) => h.UpdateBorderColor(),
 				[Microsoft.Maui.Controls.Compatibility.Layout.IsClippedToBoundsProperty.PropertyName] = (h, _) => h.UpdateClippedToBounds(),
-				[Frame.ContentProperty.PropertyName] = (h, _) => h.UpdateContent()
+				[Frame.ContentProperty.PropertyName] = (h, _) => h.UpdateContent(),
+				[VisualElement.OpacityProperty.PropertyName] = (h, v) => h.UpdateOpacity(v)
```

Comment:
So, this is actually a result of an Android optimization we have in the code so that certain sets of properties will update as a batch. 

Because these handlers don't inherit from `ViewHandler` this code never runs

https://github.com/dotnet/maui/blob/9ee62c1b125424ea54c99661de8be951f21203a3/src/Core/src/Handlers/View/ViewHandler.cs#L126-L135

Reply:
I think a possible overall fix here would be to add a mapper property that calls the `Initialize` Method

`["Initialize"] = (h, v) => h?.Initialize(v)`